### PR TITLE
CITA doc minor fixes

### PR DESCRIPTION
**Pull request by ashchan**
* Update link to CITA intro doc
* Fix language names(case sensitive)

### CODE REVIEW

**Review by zhouyun-zoe — APPROVED** [review]

File: docs/docs-intro.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ sidebar_label: CITAHub 文档概览
 
 ## CITA | 区块链内核
 
-[![](https://img.shields.io/badge/CITA-Documents-green.svg)](https://cryptape.github.io/cita/)
+[![](https://img.shields.io/badge/CITA-Documents-green.svg)](https://docs.citahub.com/zh-CN/cita/cita-intro)
```
Comment (kaikai1024) [inline docs/docs-intro.md:13]: @zhouyun-zoe Perhaps it's better that CITA also uses the Chinese version doc link.

Comment (ashchan) [reply docs/docs-intro.md:13]: Ideally when all are updated we have links to zh-CN docs from Chinese pages, so for us-EN from English pages.

Comment (kaikai1024) [reply docs/docs-intro.md:13]: > Ideally when all are updated we have links to zh-CN docs from Chinese pages, so for us-EN from English pages.

Agree with that. I mean Chinese as the first choice  maybe is better for users, check this place of CITA: 
screenshot from 2019-02-28 13-40-26

While it's English pages, so i have some doubts.

Comment (ashchan) [reply docs/docs-intro.md:13]: Indeed. Need more polish work to make this better.

**Pull request merged**